### PR TITLE
[50448] Adjusted line breaking settings for header label

### PR DESCRIPTION
**Pull request by jonasofrich**
Adjusted line height for top header in messages view

iPhone SE gen 2
Screenshot 2020-12-02 at 09 55 03

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19.iOS/Views/MessagePage/MessagePageViewController.cs [hunk 1]
```diff
@@ -67,7 +67,7 @@ public override void ViewWillDisappear(bool animated)
 
         private async void SetStyling()
         {
-            StyleUtil.InitLabelWithSpacing(Label, StyleUtil.FontType.FontBold, MESSAGES_HEADER, 1.14, 22, 34);
+            StyleUtil.InitLabelWithSpacing(Label, StyleUtil.FontType.FontBold, MESSAGES_HEADER, 0.9, 22, 34);
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19.iOS/Views/MessagePage/MessagePageViewController.cs:70]: 0.9 ? :)

**Review by pavshr — COMMENTED** [review]
Nice!

File: NDB.Covid19/NDB.Covid19.iOS/Views/MessagePage/MessagePageViewController.cs (hunk 1 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19.iOS/Views/MessagePage/MessagePageViewController.cs:70]: The descender on the G in the heading is cut if it's set any higher :/

**Comment by nc-kano** [comment]
Line spacing seems a bit... large. The whole new line can match there. Maybe we should make it smaller. But maybe it is ok. For now we can go with it and testers always can create a bug for it.

File: NDB.Covid19/NDB.Covid19.iOS/Views/MessagePage/MessagePageViewController.cs (hunk 1 above)
Comment (pavshr) [reply NDB.Covid19/NDB.Covid19.iOS/Views/MessagePage/MessagePageViewController.cs:70]: I can see some pixels are cut there as well with 0.9. We need to rework it.

**Review by pavshr — CHANGES_REQUESTED** [review]
Some changes here should be made to line height

**Comment by jonasofrich** [comment]
Screenshot 2020-12-02 at 12 56 47

This is SE Gen 2 with font size set to max.

**Review by nc-kano — APPROVED** [review]

**Review by pavshr — APPROVED** [review]

**Pull request merged**